lib/time.zsh

```diff
@@ -54,12 +54,15 @@ typeset -g geometry_time_color
 # from https://github.com/sindresorhus/pretty-time-zsh
 prompt_geometry_seconds_to_human_time() {
   local total_seconds=$1
+  local long_format=$2
+
   local days=$(( total_seconds / 60 / 60 / 24 ))
   local hours=$(( total_seconds / 60 / 60 % 24 ))
   local minutes=$(( total_seconds / 60 % 60 ))
   local seconds=$(( total_seconds % 60 ))
 
-  if $PROMPT_GEOMETRY_GIT_TIME_SHORT_FORMAT; then
+  # It looks redundant but it seems it's not
+  if [[ $long_format == true ]]; then
```

Comment:
Not sure tbh. I tried `[[ $long_format ]]` and it messed up. Gonna take a look later today to see what was going on.